panels/site_sections/schedule.py

```diff
@@ -11,7 +11,6 @@ def index(self, session, message=''):
         else:
             raise HTTPRedirect("internal")
 
-    @cached
```

Comment:
You gotta re-cache this!

Reply:
I can’t figure out why it was cached in the first place! But I’ll add the cache back 😅

Reply:
I think it's cached because people would view it a lot on-site, hitting our servers real bad. There's probably better ways to handle that than caching it, but we've been bitten many times by the "oh we'll remember to enable caching before we go onsite" pattern.